tests/monotouch-test/Foundation/NSDictionary2Test.cs

```diff
@@ -557,6 +557,7 @@ public void IEnumerableTest ()
 			Assert.AreEqual (2, c, "Enumerator Count");
 		}
 
+#if !MONOMAC // TODO: Messaging is internal on mac
```

Comment:
Missed that it had a separate version in monotouch-test.  Should be an easy fix :)